### ISSUE TITLE
Disable suspend on lid close

### ISSUE THREAD

**Issue by ahmadsocrat** [stockmind/dell-xps-9560-ubuntu-respin#15]
Hi Guys,

I now have Ubuntu 18.04 on my Dell XPS 9560 and it's working fine.

One of the problems that I currently have is that I am unable to stop the laptop from sleeping when I close the lid, e.g. when I have an external screen connected or docking station.

Any idea how this can be done?

**Comment by damageboy** [comment]
I've failed so far in getting this to work through /etc/systemd/logind.conf although there are settings for this there.

But I can make it work by closing the lid while not connected, then connecting the docking station power cable

**Comment by breakanalysis** [comment]
does it still occur if chosing external monitor as primary one before closing lid?

**Comment by ahmadsocrat** [comment]
Yes, it doesn't occur only if I totally disable the laptop's display.

Kind Regards / مع تحياتي

أحمد عمرو
Ahmad Amr
